post-office-stub/src/README.md

```diff
@@ -0,0 +1,10 @@
+# Gov Notify stub
```

Comment:
Thanks for adding this! Think line 1 might have been left in by mistake?

A couple of things that I think we could add that would be helpful to anyone picking this up that doesn't have context would be 
1. How to run this thing - is it the normal deploy steps? When might someone want to use it and how would they hook it up to their custom stack to do so?
2. An example request payload

Reply:
Fixed line 1 and added link to confluence page which has more info - in terms of deployment yes it is the normal deploy steps